webrtc.html

```diff
@@ -1992,27 +1992,25 @@
 
         <dl class="idl" data-merge="RTCIceCandidateInit" title=
         "interface RTCIceCandidate">
-          <dt>Constructor (optional RTCIceCandidateInit candidateInitDict)</dt>
+          <dt>Constructor (RTCIceCandidateInit candidateInitDict)</dt>
 
           <dd>The <dfn id=
           "dom-icecandidate"><code>RTCIceCandidate()</code></dfn> constructor
-          takes an optional dictionary argument, <var>candidateInitDict</var>,
+          takes a dictionary argument, <var>candidateInitDict</var>,
           whose content is used to initialize the new
-          <code><a>RTCIceCandidate</a></code> object. If a dictionary key is
-          not present in <var>candidateInitDict</var>, the corresponding
-          attribute will be initialized to null. If the constructor is run
-          without the dictionary argument, all attributes will be initialized
-          to null.</dd>
+          <code><a>RTCIceCandidate</a></code> object. When constructed, values
+          for <var>candidate</var> and either <var>sdpMid</var>
+          or <var>sdpMLineIndex</var> MUST be provided.</dd>
```

Comment:
It might be quicker now to generate a PR rather than continue discussing here, given that this has been merged.